Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:latest
+RUN apt update -y && apt install hugo -y
```

Comment:
Any reason we are making our own instead of using the [official](https://hub.docker.com/r/klakegg/hugo/) container?

Reply:
According to the [Hugo website](https://gohugo.io/getting-started/installing/)

> We currently do not offer official Hugo images for Docker, but we do recommend these up to date distributions: https://hub.docker.com/r/klakegg/hugo/

So I ended up making our own Dockerfile